src/components/panels/collapsible-panel/README.md

```diff
@@ -33,21 +33,22 @@ The component can be used in a controlled or uncontrolled manner. Please refer t
+| ----------------------- | -------- | :------------------------------------------------------: | --------------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+| `header`                | `node`   |                            ✅                            | -                     | -         | The title being rendered at top left of the panel                                                                                         |
+| `secondaryHeader`       | `node`   |                                                          | -                     | -         | A secondary header for the panel (only pass if needed)                                                                                    |  |
+| `onToggle`              | `func`   | (required only if controlled as in `isClosed` is passed) | -                     | -         | function to be triggered whenever the user clicks the top area to collapse the panel's content                                            |
+| `isClosed`              | `bool`   |                                                          | -                     | false     | Indicates if the panel's content should be collapsed or shown                                                                             |
+| `isSticky`              | `bool`   |                                                          | -                     | false     | Makes the panel's header sticky in regards to the page's scroll                                                                           |
+| `description`           | `String` |                                                          | -                     |           | If passed will be shown below the title as more information regarding the panel                                                           |
+| `isDisabled`            | `bool`   |                                                          | -                     | false     | Disables the panel and all interactions with it                                                                                           |
+| `children`              | `node`   |                                                          | -                     | -         | The actual content rendered inside the panel                                                                                              |
+| `hideExpansionControls` | `bool`   |                                                          | -                     | -         | Controls the visibility of the expansion controls on the left                                                                             |
```

Comment:
I'm not sure about this prop name. @dferber90 any ideas?

Reply:
It's an attempt to align it with some inputs which also have an option to be expanded. The intend was to have transferable vocabulary also across components of different groups. Let me know if you come up with a better name and the reasoning for it.

Reply:
but we no longer have any other props called `hideExpansionControls`, they were changed [here](https://github.com/commercetools/ui-kit/pull/389).

Reply:
Your call. Gimme a name :)

Reply:
Well in this case you are expanding or collapsing... maybe the name is correct after all 🤷‍♂️ 

Just a note this is just in case the component is disabled right? So we could assume that you set it whenever is disabled meaning that the panel is collapsed so now you can only expand 🤔

Reply:
I don't really have strong opinions on it. hideExpansionControls is fine by me. 🤷‍♂️

Reply:
I will wait for the @dferber90 to give his feedback for this.

Reply:
I'm fine with this name 👍 

We used more specific names (`defaultExpandMultilineText`, `defaultExpandLanguages`) in `LocalizedMultilineTextInput` as there are two things which can be expanded/collapsed. Here it's only one thing, so `hideExpansionControls` seems fine. On a sidenote, `LocalizedMultilineTextInput` only accepts `hideLanguageExpansionControls` as the multiline expansion controls can't be hidden (yet).

<details>
<sumary>Sidenote</summary>

`LocalizedMultilineTextInput` always acts as an uncontrolled component regarding its expansion states (it doesn't accept the state from the parent). Therefore it doesn't accept props like `expandMultilineText` or `expandLanguages` (or `isMultilineTextExpanded` and `areLanguagesExpanded`).

For the component in this PR, I slightly dislike that `hideExpansionControls` and `isClosed` don't use the same terminology (open/closed vs expanded/collapsed). We could do `hideExpansionControls` and `isExpanded` so that they use the same terminology for the same concept which shows that the props belong together. But that's probably not worth the refactoring as we'd also need to flip the default. So let's just keep `isClosed` and add `hideExpansionControls`.

</details>